CHANGES.rst

```diff
@@ -8,6 +8,7 @@ Unreleased
 
 * ``config_interpolation`` parameter for :class:`.Reddit` supporting basic and
   extended modes.
+* :meth:`.Redditors.user_summaries`
```

Comment:
Please go more in-depth on this change. Explain the change, what does the method do?

Reply:
Oh, right, certainly.